insitu/views/product_requirement.py

```diff
@@ -24,6 +25,11 @@ class BaseProductRequirementAdd(CreatedByMixin, LoggingProtectedCreateView):
     permission_classes = (IsAuthenticated, )
     permission_denied_redirect = reverse_lazy('requirement:list')
 
+    def form_valid(self, form):
```

Comment:
Set message = None on __BaseProductRequirementAdd__ class. Define it in __ProductRequirementAdd__ and __ProductGroupRequirementAdd__. Use message in the form_valid method defined in __BaseProductRequirementAdd__ class. The message for  __ProductGroupRequirementAdd__ should be something like *"The relations between the product group's products and requirement were created succesfully!"*